src/use_case/add_exercise/AddExerciseInputData.java

```diff
@@ -1,5 +1,25 @@
 package use_case.add_exercise;
 
+import java.util.ArrayList;
+
 public class AddExerciseInputData {
+    final private int id;
+
+    final private String exercise_name;
```

Comment:
hello! sorry, commented on the wrong spot i think. check comment on add exercise data access interface